app/policies/project_policy.rb

```diff
@@ -21,7 +21,7 @@ class TranslateScope < Scope
         :create_subjects_export,
         :create_workflows_export,
         :create_workflow_contents_export,
-        :retire_subjects, with: WriteScope
+        :retire_subjects, :unretire_subjects, with: WriteScope
```

Comment:
this should probably have associated spec entries for the different authorization levels (types of user), e.g. for collaborator https://github.com/zooniverse/panoptes/blob/61d184848bfe367f4536ddbe3054988c3e734741/spec/policies/project_policy_spec.rb#L177